cypress/utils/utils.ts

```diff
@@ -1813,6 +1813,21 @@ export function getCommandOutput(command: string): Cypress.Chainable<Cypress.Exe
     });
 }
 
+export function validateCliVersion(expectedMtaVersion: string): void {
```

Comment:
We can rename it validateMTAVersionThroughCLI

Reply:
Done